src/OpenDiffix.Core/CommonTypes.fs

```diff
@@ -217,6 +217,7 @@ type AggregationBucket =
     Group: Row
     Aggregators: IAggregator array
     ExecutionContext: ExecutionContext
+    LowCount: bool
```

Comment:
I think this field should be an `IAggregator` object. That way, the low-count property can be both evaluated and merged with another bucket's property.

Reply:
The `IAggregator` can be found in `Aggregators`. We may pass an index or make it first like you suggested.
The reason for the constant boolean is that for LED we don't want a change in `LowCount` mid-way because it will make the algorithm order dependent, although extremely unlikely to be reproducible for an attack.

Reply:
This is a concern specific to the LED hook, which shouldn't spill into the overall design.
Will it be too complex to remember the initial atribute value inside the hook? If yes, then keep this as is.

Reply:
Ok, it can be memo'd inside the LED hook, not a problem. I'm still thinking on how do we treat multiple LCF's in scope.

Maybe we raise during compilation?

Reply:
I don't currently see the utility of a scenario in which we can have more than one, so raising would be fine for now.

Reply:
No utility whatsoever, but we limit the freehand queries that can be called via reference CLI.